src/core/tests/type_prop/unary_elementwise.cpp

```diff
@@ -2,21 +2,25 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "common_test_utils/test_assertions.hpp"
 #include "common_test_utils/type_prop.hpp"
 #include "openvino/op/negative.hpp"
 
 using namespace std;
 using namespace ov;
+using namespace testing;
```

Comment:
```suggestion
using testing::HasSubstr;
```
If required for `HasSubstr`

Reply:
Ok, done